sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample1_DetectEntireSeriesAnomaly.md

```diff
@@ -0,0 +1,73 @@
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateAnomalyDetectorClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+
+var endpointUri = new Uri(endpoint);
+var credential = new AzureKeyCredential(apiKey);
+
+AnomalyDetectorClient client = new AnomalyDetectorClient(endpointUri, credential);
+```
+
+## Load time series and create DetectRequest
+
+You could download our [sample data][SampleData], read in the time series data and add it to a `DetectRequest` object.
+
+Call `File.ReadAllLines` with the file path and create a list of `TimeSeriesPoint` objects, and strip any new line characters. Extract the values and separate the timestamp from its numerical value, and add them to a new `TimeSeriesPoint` object.
+
+Make a `DetectRequest` object with the series of points, and `TimeGranularity.Daily` for the granularity (or periodicity) of the data points.
+
+```C# Snippet:ReadSeriesData
+string datapath = "<dataPath>";
+
+List<TimeSeriesPoint> list = File.ReadAllLines(datapath, Encoding.UTF8)
+    .Where(e => e.Trim().Length != 0)
+    .Select(e => e.Split(','))
+    .Where(e => e.Length == 2)
+    .Select(e => new TimeSeriesPoint(DateTime.Parse(e[0]), float.Parse(e[1]))).ToList();
+
+DetectRequest request = new DetectRequest(list, TimeGranularity.Daily);
+```
+
+## Detect anomalies of the entire series
+Call the client's `DetectEntireSeriesAsync` method with the `DetectRequest` object and await the response as an `EntireDetectResponse` object. Iterate through the response's `IsAnomaly` values and print any that are true. These values correspond to the index of anomalous data points, if any were found.
+
+```C# Snippet:DetectEntireSeriesAnomaly
+Console.WriteLine("Detecting anomalies in the entire time series.");
+
+EntireDetectResponse result = await client.DetectEntireSeriesAsync(request).ConfigureAwait(false);
+
+if (result.IsAnomaly.Contains(true))
+{
+    Console.WriteLine("An anomaly was detected at index:");
+    for (int i = 0; i < request.Series.Count; ++i)
+    {
+        if (result.IsAnomaly[i])
+        {
+            Console.Write(i);
+            Console.Write(" ");
+        }
+    }
+    Console.WriteLine();
+}
+else
+{
+    Console.WriteLine(" No anomalies detected in the series.");
+}
+```
+To see the full example source files, see:
+
+* [Detect Entire Series Anomaly](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/anomalydetector/Azure.AI.AnomalyDetector/tests/samples/Sample1_DetectEntireSeriesAnomaly.cs)
+
+[README]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/anomalydetector/Azure.AI.AnomalyDetector/README.md
+[SampleData]: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/data/request-data.csv
```

Comment:
Incorrect link: you moved `samples/data/request-data.csv` file under tests.

Reply:
We would need to fix the link for other two files as well.

Reply:
Fixed.

---
In reply to: [483867814](https://github.com/Azure/azure-sdk-for-net/pull/14579#discussion_r483867814) [](ancestors = 483867814)

Reply:
Fixed.

---
In reply to: [483868090](https://github.com/Azure/azure-sdk-for-net/pull/14579#discussion_r483868090) [](ancestors = 483868090)